src/DataTable/utils/queryParams.js

```diff
@@ -337,6 +337,17 @@ function getSubFilter(
             fieldVal.toLowerCase()
           );
         };
+  } else if (ccSelectedFilter === "notContains") {
+    return qb
+      ? qb.notContains(stringFilterValue.replace(/_/g, "\\_")) //filter using qb (aka we're backend connected)
+      : fieldVal => {
+          //filter using plain old javascript (aka we've got a local table that isn't backend connected)
+          if (!fieldVal || !fieldVal.toLowerCase) return false;
+          return (
+            fieldVal.toLowerCase().replace(filterValLower, "") ===
+            fieldVal.toLowerCase()
+          );
+        };
```

Comment:
Seems like both the "contains" and the "notContains" apply exactly the same cleaning. Couldn't we just merge the two into a single else if block?